examples/aliens.py

```diff
@@ -244,7 +244,7 @@ def main(winstyle=0):
     pg.init()
     if pg.mixer and not pg.mixer.get_init():
         print("Warning, no sound")
-        pg.mixer = None
+        # pg.mixer = None
     pg.mixer = None
```

Comment:
Thanks for finding this. 

However, I think you commented out the wrong line. Instead I think line 248 should be removed.